new/components/manager/scripts/configure_manager.sh

```diff
@@ -7,10 +7,14 @@ function _post_provider_context() {
     export PROVIDER_CONTEXT_FILE=$(ctx download-resource "components/manager/config/provider_context")
     export PROVIDER_CONTEXT_DATA=$(cat $PROVIDER_CONTEXT_FILE)
 
+    sed_delimiter=$'\001'
+    cloudify_agent="$(ctx --json-output node properties cloudify.cloudify_agent)"
+    export PROVIDER_CONTEXT_DATA=$(echo "${PROVIDER_CONTEXT_DATA}" | sed s"${sed_delimiter}{{ cloudify_agent }}${sed_delimiter}${cloudify_agent//\\/\\\\}${sed_delimiter}")
+
     ctx instance runtime_properties manager_provider_context "$PROVIDER_CONTEXT_DATA"
 
     ctx logger info "Posting Provider Context..."
-    curl --fail --silent --request POST --data @${PROVIDER_CONTEXT_FILE} http://localhost/api/v2/provider/context --header "Content-Type:application/json" >/dev/null
+    curl --fail --silent --request POST --data "${PROVIDER_CONTEXT_DATA}" http://localhost/api/v2/provider/context --header "Content-Type:application/json" >/dev/null
```

Comment:
I think this is actually from the mods I made following @nir0s 's initial new manager deployment which curled this up to the manager. I believe it is actually fully bypassed now, but would need to check with @nir0s 

The function this bit of code is in is not run any more, from memory.